public/index.html

```diff
@@ -0,0 +1,215 @@
+          <li class="mapLi"><button class="mapButtons" data-content="february" id="february" type="button" name="button">february</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="march" id="march" type="button" name="button">march</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="april" id="april" type="button" name="button">april</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="may" id="may" type="button" name="button">may</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="june"id="june" type="button" name="button">june</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="july" id="july" type="button" name="button">july</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="august" id="august" type="button" name="button">august</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="september" id="september" type="button" name="button">september</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="october" id="october" type="button" name="button">october</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="november" id="november" type="button" name="button">november</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="december"id="december" type="button" name="button">december</button></li>
+        </ul>
+
+
+      <section id="mapBox" style="background-image: url({{map}})">
+        <canvas id="coffeeData" width="300" height="525"></canvas>
+      </section>
+      </section>
+    </section>
+    <section id="brewHead">
+      <h1 id="brewH1">How we Brew it</h1>
+      <img id ="straightArrowLeft" class="arrows" src="/media/images/straightArrowLeft.png" alt="arrow" />
+    </section>
+    <section class="coffeePages" id="brewMethods">
+      <section id="methodsContainer">
+        <section class="methodEls" id="brewImage">
+          <img src="{{brewPic}}" alt="not found" />
+        </section>
+        <section class="methodEls" id="brewSteps">{{brewInstructions}}</section>
+      </section>
+    </section>
+  </section>
+  </script>
+
+
+
+  </head>
+  <body>
+    <select id="coffeeCountry" name="Country">
+      <option class="menuOptions" value="nicaraguaPage.json">Nicaragua</option>
+      <option class="menuOptions" value="hondurasPage.json">Honduras</option>
+      <option class="menuOptions" value="guatemalaPage.json">Guatemala</option>
+
+    </select>
+    <header class="tab header">
+      <a href="/">
+      <section class="logo">
+        <h1 id="origin1H1">origin1<span id="cs"><br>coffee + source</br></span></h1>
+      </section>
+      <div class="navCollapsed">
+        <div id="navButton">
+        </a>
+          <img src="media/images/arabica-beanUse.png"/>
```

Comment:
I like this whole menu drop-out, but the coffee bean logo should stay positioned in the same place when clicked.